langchain/prompts/base.py

```diff
@@ -9,6 +13,16 @@
 }
 
 
+def cleanup_prompt_dict(prompt_dict: Dict) -> None:
```

Comment:
why is this neccessary? i tried deleting it and the tests you added (thanks for tht btw) still pass

imo if this is not neccessary we should not include, if it is lets add a test

Reply:
this is for readibility. keys that are None are still saved - didn't want those showing up in the yaml file

Reply:
lets just keep them in. i dont think it hurts readability too much to have them in, and would prefer to avoid unneccessary logic. if people start to complain, then we can add this in :)

Reply:
bump